geometry_msgs/Point[] clusters